mlrun/model.py

```diff
@@ -281,6 +378,18 @@ def __init__(
 
 
 class BaseMetadata(ModelObj):
+    _fields_to_strip = [
+        # "tag", # should tag be removed as well?
+        "hash",
+        "namespace",
+        "project",
+        "labels",
+        "annotations",
+        "categories",  # should categories be removed as well?
```

Comment:
I don't think the `categories` should be removed - they are describing the object, like name or labels, so I don't see a reason to remove them.